src/org/exist/backup/FileSystemBackupDescriptor.java

```diff
@@ -33,9 +35,13 @@
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.util.Properties;
+import java.util.stream.Stream;
 
 
 public class FileSystemBackupDescriptor extends AbstractBackupDescriptor {
+
+    private final static Logger LOG = LogManager.getLogger();
```

Comment:
Don't we need a class here for the logger? e.g. `getLogger(AbstractBackupDescriptor.class)`?

Reply:
AFAIK this is not needed (anymore). in the era of log4j 1.2.x is was indeed required for some time.

see https://logging.apache.org/log4j/2.x/log4j-api/apidocs/org/apache/logging/log4j/LogManager.html#getLogger()